src/linear_algebra/affine_space.lean

```diff
@@ -55,16 +55,32 @@ include S
 of points. -/
 def vector_span (s : set P) : submodule k V := submodule.span k (vsub_set V s)
 
+/-- The `vsub_set` lies within the `vector_span`. -/
+lemma vsub_set_subset_vector_span (s : set P) : vsub_set V s ⊆ vector_span k V s :=
+submodule.subset_span
+
+/-- Each pairwise difference is in the `vector_span`. -/
+lemma vsub_mem_vector_span {s : set P} {p1 p2 : P} (hp1 : p1 ∈ s) (hp2 : p2 ∈ s) :
+  p1 -ᵥ p2 ∈ vector_span k V s :=
+begin
+  rw ←submodule.mem_coe,
+  exact set.mem_of_mem_of_subset (vsub_mem_vsub_set V hp1 hp2)
+                                 (vsub_set_subset_vector_span k V s)
+end
+
 /-- The points in the affine span of a (possibly empty) set of
-points. Use `affine_span` instead to get an `affine_subspace k V P`,
-if the set of points is known to be nonempty. -/
+points. Use `affine_span` instead to get an `affine_subspace k V P`. -/
 def span_points (s : set P) : set P :=
 {p | ∃ p1 ∈ s, ∃ v ∈ (vector_span k V s), p = v +ᵥ p1}
 
 /-- A point in a set is in its affine span. -/
 lemma mem_span_points (p : P) (s : set P) : p ∈ s → p ∈ span_points k V s
 | hp := ⟨p, hp, 0, submodule.zero_mem _, (zero_vadd V p).symm⟩
 
+/-- A set is contained in its `span_points`. -/
+lemma span_points_mono (s : set P) : s ⊆ span_points k V s :=
```

Comment:
```suggestion
lemma subset_span_points (s : set P) : s ⊆ span_points k V s :=
```
`span_points_mono` would be `s \subset t → span_points k V s \subset span_points k V t`.